app/views/admin/products_v3/_product_row.html.haml

```diff
@@ -10,6 +10,8 @@
   = f.text_field :sku, 'aria-label': t('admin.products_page.columns.sku')
   = error_message_on product, :sku
 %td.multi-field{ 'data-controller': 'toggle-control', 'data-toggle-control-match-value': 'items' }
+  = f.hidden_field :variant_unit
+  = f.hidden_field :variant_unit_scale
```

Comment:
I'm just wondering about a new approach. Could we simplify the frontend preview by ignoring the actual data model and just say: `variant_unit_value + " " + variant_unit`? That's probably too simple and doesn't resemble the real display in all cases...

Reply:
I'm not sure I follow, but I think the answer will as you predicted: 
> probably too simple and doesn't resemble the real display in all cases...